pkg/graph/resolvers/trb_guidance_letter_recommendation.go

```diff
@@ -128,28 +130,16 @@ func DeleteTRBGuidanceLetterRecommendation(ctx context.Context, store *storage.S
 		}
 	}
 
-	// deleting the given recommendation and updating the other recommendations can be done concurrently
-	// ideally, we'd do this in a single transaction, but our code doesn't support that at this time - see Note [Database calls from resolvers aren't atomic]
-	errGroup := new(errgroup.Group)
-	var deletedRecommendation *models.TRBGuidanceLetterRecommendation // declare this outside the function we pass to errGroup.Go() so we can return it
-
-	errGroup.Go(func() error {
-		deletedRecommendation, err = store.DeleteTRBGuidanceLetterRecommendation(ctx, id)
-		if err != nil {
-			return err
-		}
-		return nil
-	})
-
-	errGroup.Go(func() error {
-		_, err := store.UpdateTRBGuidanceLetterRecommendationOrder(ctx, trbRequestID, newOrder)
-		if err != nil {
-			return err
-		}
-		return nil
-	})
+	deletedRecommendation, err := store.DeleteTRBGuidanceLetterRecommendation(ctx, id)
```

Comment:
Perhaps we should use `WithTransaction` here to wrap these operations so they're all atomic?

Reply:
will take a look! thanks!

Reply:
agreed - i'll put something up shortly! thanks

Reply:
i put a PR up so the differences were clear if you wouldn't mind taking a look! https://github.com/CMS-Enterprise/easi-app/pull/2867